CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,76 @@
+underrepresented in the communities that we serve, or are oppressed by
+structural sexism, racism, classism, ableism, homophobia, xenophobia, and
+transphobia. This includes but is not limited to: women, people of color,
+people with disabilities, people with prior military service, formerly
+incarcerated people, people who identify as LGBTQIA+, indigenous peoples, first
+and second generation immigrants, and people from low-income families.
+
+By contributing to the development of this tool you agree to abide by this
+code of conduct and cooperate with the Consumer Reports staff who enforce it.
+
+### Expected Behavior
+
+Individuals are expected to behave appropriately and professionally during
+their participation in this project. In particular, all individuals 
+should actively avoid intentionally or unintentionally participating in the 
+harassment of individuals. Harassment includes offensive comments related to 
+gender, sexual orientation, disability, mental health, neurotype, age, 
+physical appearance, body, race, ethnicity, nationality or religion; 
+deliberate or repeated misgendering; sexual images in public spaces;
+intimidation; stalking; following; harassing photography or recording;
+sustained disruption in meetings, lectures or other spaces; physical contact
+and simulated physical contact (e.g. emoji or textual descriptions) without
+consent; and unwelcome sexual attention. Consent must be freely given,
+enthusiastic, informed, specific, and reversible affirmation of a request.
+
+Individuals are expected to demonstrate kindness and care when communicating
+with one another. In recognizing the importance of diverse perspectives and
+expertise, Bocoup expects individuals to listen actively, and to interact with
+respect and humility.
+
+### Consequences of Unacceptable Behavior
+
+Consumer Reports in the course of enforcing this Code of Conduct may, at its own
+discretion, ask an individual to stop some behavior, warn the individual of
+their violation, or further sanction the individual. 
+
+### Reporting Unacceptable Behavior
+
+If you would like to report a violation of this Code of
+Conduct, please contact Ginny Fahs at ginny.fahs.consultant@consumer.org. Any further 
+communication will be determined with your consent. If
+another individual asks you to stop engaging in some behavior in the course of
+your work with Consumer Reports, you should comply immediately, and are encouraged to
+contact Ginny Fahs for further guidance.
+
+Consumer Reports prioritizes marginalized people’s safety over the comfort of individuals
+in positions of privilege. Consumer Reports reserves the right not to act on complaints
+regarding “reverse-isms”, reasonable communication of boundaries, complaints
+about tone premised on stereotypes, or criticism of racist, sexist, classist or
+otherwise oppressive behavior.
+
+## Thank you and Credits
+
+This code of conduct is influenced by the [Processing
```

Comment:
This code of conduct is based off the [Bocoup code of conduct](https://bocoup.com/code-of-conduct)
which was heavily influenced by the [Processing